global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "3.1.300"
+    "version": "5.0.100-rc.1.20367.2"
```

Comment:
should we explicitly set rollForward ?

Reply:
I don't know. I have used that setting.

Is it required? I haven't had trouble with installing newer versions yet.